modules/nf-core/parabricks/indexgvcf/tests/main.nf.test.snap

```diff
@@ -0,0 +1,146 @@
+{
+    "human - gvcf.gz": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "test"
+                        },
+                        "versions.yml:md5,a6c2622b9fd9223dbc349c2a27e9a2e4"
+                    ]
+                ],
+                "1": [
+                    "versions.yml:md5,a6c2622b9fd9223dbc349c2a27e9a2e4"
+                ],
```

Comment:
This looks funky to me, where is the index file(s) itself?

Reply:
should be right below that, no? called gvcf_index

Reply:
I see a meta field and `versions.yml`, there is no `.idx` nor `.tbi` listed with an md5sum

![image](https://github.com/user-attachments/assets/24900422-109a-486d-9e82-b5ceda4c87d0)

Reply:
You are completely right - I think I found the issue.